statspersonalinfos.php

```diff
@@ -352,7 +359,7 @@ protected function getData($layers)
 
             case 'currency':
                 $this->_titles['main'] = $this->trans('Currency distribution', array(), 'Modules.Statspersonalinfos.Admin');
-                $sql = 'SELECT c.`name`, COUNT(c.`id_currency`) AS total
+                $sql = 'SELECT c.`iso_code`, COUNT(c.`id_currency`) AS total
```

Comment:
```suggestion
                $sql = 'SELECT IFNULL(c.`name`, c.`iso_code`) as name, COUNT(c.`id_currency`) AS total
```
Something like this? or if it's not null, use a case when?

Reply:
Always empty, the column `name` is unused since PrestaShop put this field in `ps_currency_lang` table

Reply:
Since when? The module is still compatible with 1.7.1.0.
Can you add a join to the `ps_currency_lang`?

Reply:
Since PrestaShop 1.5 I think 🤣

Reply:
Done